src/cider/type/plan/Expr.cpp

```diff
@@ -30,6 +30,10 @@ JITExprValue& Expr::codegen(JITFunction& func) {
   UNREACHABLE();
   return expr_var_;
 }
+
+JITExprValue& Expr::codegen(JITFunction& func, context::CodegenContext& context) {
```

Comment:
please see this discussion thread https://github.com/intel/BDTK/pull/280#discussion_r1057155512